contracts/base/ModuleManager.sol

```diff
@@ -40,6 +40,46 @@ abstract contract ModuleManager is SelfAuthorized, Executor, GuardManager {
         }
     }
 
+    /**
+     * @notice Runs pre-execution checks for module transactions if a guard is enabled.
+     * @param to Target address of module transaction.
+     * @param value Ether value of module transaction.
+     * @param data Data payload of module transaction.
+     * @param operation Operation type of module transaction.
+     * @return guard Guard to be used for checking.
+     * @return guardHash Hash returned from the guard tx check.
+     */
+    function preModuleExecution(
+        address to,
+        uint256 value,
+        bytes memory data,
+        Enum.Operation operation
+    ) internal returns (address guard, bytes32 guardHash) {
+        guard = getGuard();
+
+        // Only whitelisted modules are allowed.
+        require(msg.sender != SENTINEL_MODULES && modules[msg.sender] != address(0), "GS104");
+
+        if (guard != address(0)) {
+            guardHash = Guard(guard).checkModuleTransaction(to, value, data, operation, msg.sender);
+        }
+    }
+
+    /**
+     * @notice Runs post-execution checks for module transactions if a guard is enabled.
+     * @param guardHash Hash returned from the guard during pre execution check.
+     * @param success Boolean flag indicating if the call succeeded.
+     * @param guard Guard to be used for checking.
+     * @dev Emits event based on module transaction success.
+     */
+    function postModuleExecution(bytes32 guardHash, bool success, address guard) internal {
```

Comment:
Sorry for being a bit OCD :see_no_evil:... for me it would make slightly more sense for this to be:

```suggestion
    function postModuleExecution(address guard, bytes32 guardHash, bool success) internal {
```

So that the parameters are grouped slightly more logically (and in the same order that they are used/created in the higher level functions).

Not blocking merging the PR.

Reply:
Done 👍🏾